hungry_toaster_description/urdf/hungry_toaster_description.urdf

```diff
@@ -0,0 +1,25 @@
+<robot name="hungryToaster">
+  <link name="base_link">
+    <visual>
+      <origin rpy="0 0 0" xyz="0.0 0 0.0"/>
+      <geometry>
+        <mesh filename="package://hungryToaster_description/meshes/hungry-toaster-meters.stl"/>
+      </geometry>
+    </visual>
+  </link>
+
+  <joint name="base_link_to_laser" type="fixed">
+    <parent link="base_link"/>
+    <child link="Laser"/>
+    <origin xyz="0 0 1.0"/>
```

Comment:
we'll probably have to update the origin (and add rpy so that the laser orientation is correct) -- but I'm sure you already know that.